deepmd/entrypoints/test.py

```
@@ -303,10 +327,11 @@ def test_ener(
     if dp.has_spin:
         data.add("spin", 3, atomic=True, must=True, high_prec=False)
         data.add("force_mag", 3, atomic=True, must=False, high_prec=False)
-    if dp.has_hessian:
-        data.add("hessian", 1, atomic=True, must=True, high_prec=False)
 
     test_data = data.get_test()
+    find_energy = test_data.get("find_energy")
```

Ruff (0.8.2)

Local variable `find_energy` is assigned to but never used

Remove assignment to unused variable `find_energy`

(F841)